src/server/blankett/components/Header.tsx

```diff
@@ -21,7 +22,7 @@ function Header(props: HeaderProps) {
         </div>
       </div>
       <div className={'tittel-og-personinfo'}>
-        <h2 className="tittel">{tittel.toUpperCase()}</h2>
+        <h2 className="tittel">{startMedStorBokstav(tittel)}</h2>
```

Comment:
Hva med å bruke overskrifter uten å endre? 

Risikerer vi å endre tekst som skal ha stor bokstav i tittel (egennavn/sted?)

Reply:
Godt poeng!

Reply:
For vår del går nok begge deler fint.

Reply:
Sjekket noen av brevene våre i Sanity, alle jeg sjekket har tittel som starter med stor bokstav. Så tenker at det for vår del er greit å kun bruke tittel vi sender med

Reply:
Sa lenge ingen brev har flettefelt i overskrifter av type "Hei [navn med stor forbokstav] - du er innvilget...", eller liknende så skal det jo gå greit. Jeg har bare ikke helt oversikt. Kanskje høre med brevekspert og BAKS om de tenker dette er en risiko?

Reply:
Har gått bort fra å endre til stor forbokstav og små ellers